### ISSUE TITLE
Remove (or have an option to remove) null/None values in the serialized content

### ISSUE THREAD

**Issue by travis-leith** [Zaid-Ajaj/Hawaii#22]
I am working with an API that complains about an incorrect date format on an optional field because the body contains something like `"datefield": null`

If I comment out fields that I don't use so that they don't appear in the json, then it works fine.
Would be good to be able to suppress the serialization of null/None values.

Edit: the date format error is not the only error. This happens even with optional integer fields for example.

**Comment by travis-leith** [comment]
For me this is a simple as changing a line to 
```F#
let settings = JsonSerializerSettings(Converters=[| converter |], NullValueHandling = NullValueHandling.Ignore)
```

Not sure if this would be universally acceptable so it may be better to make it a config option.

**Comment by Zaid-Ajaj** [comment]
Fixed as of v0.48 🚀 

> Not sure if this would be universally acceptable

I think it is a good default. Will add an option for it if someone needs it
